EpikCord/__init__.py

```diff
@@ -990,6 +990,18 @@ async def interaction_create(self, data):
 
         return interaction
 
+    async def guild_role_create(self,data:dict):
+        guild = self.guilds.fetch(data["guild_id"])
+        data["role"]["guild"] = guild
```

Comment:
Sorry to be so picky, but if you could change this to be `data["role"]["guild_id"] = data["guild_id"]`, because I changed how getting the Guild works, thanks